network-api/networkapi/wagtailpages/factory/blog.py

```diff
@@ -90,10 +90,10 @@ class Meta:
     title_text = Faker('sentence', nb_words=3, variable_nb_words=False)
 
 
-class BlogAuthorFactory(DjangoModelFactory):
+class ContentAuthorFactory(DjangoModelFactory):
```

Comment:
Maybe as a followup? 

Since `ContentAuthor` is being used on both fake Blog Pages and fake Article pages, can we move this to its own file so it's not so embedded in blog page generation?

Reply:
good call, I filed https://github.com/mozilla/foundation.mozilla.org/issues/6359

Reply:
Awesome!